targets/TARGET_STM/stm_spi_api.c

```
@@ -1738,7 +1738,9 @@ static bool spi_master_start_asynch_transfer(spi_t *obj, transfer_type_t transfe
             LL_SPI_Enable(SPI_INST(obj));
         }
 #endif
-        DEBUG_PRINTF("SPI: RC=%u\n", rc);
+
+        // Unfortunately there is no way to propagate the error code back up, better to print a warning than swallow it entirely
+        printf("Warning: async SPI transfer start failed in STM32 HAL, error code = %d", rc);
```

Updated this HAL to print warnings if DMA operations fail.  These warnings would have saved me a lot of time debugging the code, as I didn't know that the error codes were being swallowed.

---

What about commented out only the line 60? That will do same thing but how it was designed probably.